components/policy/resources/templates/policy_definitions/README.md

```diff
@@ -0,0 +1,22 @@
+## Policy Settings in Brave
+
+These are Brave specific group policies.
+
+Our official documentation is [available here](https://support.brave.com/hc/en-us/articles/360039248271-Group-Policy).
+
+For information about adding a new policy, please see [this page](https://source.chromium.org/chromium/chromium/src/+/main:docs/enterprise/add_new_policy.md).
+
+In a nutshell, the steps for adding a new policy in Brave look like this:
+
+1. Create a new .yaml file under `src/brave/components/policy/resources/templates/policy_definitions/BraveSoftware/`. The name of the file itself will be the policy name. Chromium uses capital casing. For example, we have a policy for disabling Brave Rewards called `BraveRewardsDisable.yaml`. The name used for matching is `BraveRewardsDisable`.
```

Comment:
prefer using workspace-based paths, like `//brave/components/...`.

Reply:
Fixed for everything 😄👍